ts/annotations/annotations.src.ts

```diff
@@ -1023,12 +1055,28 @@ merge(
          * @private
          */
         init: function (this: Highcharts.Annotation): void {
+            var plotOptions = this.chart.options.plotOptions as any,
+                defer = this.options.defer;
+
             this.linkPoints();
             this.addControlPoints();
             this.addShapes();
             this.addLabels();
             this.addClipPaths();
             this.setLabelCollector();
+
+            if (defer) {
+                // If defer duration is set use set value,
+                // else inherits from animation set in plotOptions
+                this.defer = isObject(this.options.defer) ?
+                    this.options.defer :
```

Comment:
Perhaps I miss something here but:
- if `this.options.defer` is object -> okay, use that options
- if `this.options.defer` is true -> okay, inherit defer/animation from plotOptions
- if `this.options.defer` is false -> we still inherit from plotOptions ?

Reply:
Bonus point: `this.options.defer` is already defined above in line 1059.